config/samples/kuadrant.io_v1alpha1_dnsrecord.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kuadrant.io/v1alpha1
```

Comment:
probably less valuable as really only the dns conrtroller creates these but it looks accurate